eos/constraints/B-to-P-form-factors.yaml

```diff
@@ -887,6 +887,295 @@ B_s->D_s::f_++f_0@HPQCD:2019A:
 # B_s -> K form factors #
 #########################
 # {{{
+B_s->K::form-factors[f_+,f_0,f_T]@BvDV:2023A:
```

Comment:
Can you use a more compact and readable syntax? e.g.
```
kinematics:
  [ {q2: -10.0}, {q2: -5.0}, {q2: 0.0}, {q2: 5.0},
    {q2: -10.0}, {q2: -5.0}, {q2: 5.0},
    {q2: -10.0}, {q2: -5.0}, {q2: 0.0}, {q2: 5.0} ]
```

Reply:
Lovely, requires merging #674 first.

Reply:
No no, absolutely not. This is a pure Yaml thing, look at the other constraints

Reply:
Fixed locally, for all added constraints and the renamed RBC/UKQCD constraint. We need to automatize this. Last time I failed miserably, due to floats not being stable under serialization... :-/